libraries/webFrontend.js

```diff
@@ -694,7 +693,7 @@ exports.uploadTargetContent = function (parm, objectsPath, objectInterfaceName)
 
             if (content[1][0] !== '.' && content[1][0] !== '_') {
                 if (debug)console.log(content[1]);
-                var fileTypeF = changeCase.lowerCase(content[1].split('.')[1]);
+                var fileTypeF = content[1].split('.')[1].toLowerCase();
```

Comment:
nice. not sure why we needed a module to do that before.

Reply:
change-case dropped their lowerCase function so I think they agree